sw/device/tests/usbdev_iso_test.c

```diff
@@ -116,7 +116,7 @@ static usb_testutils_streams_ctx_t stream_test;
  *   (Note that this substantially alters the timing of interactions with the
  * DPI model and will increase the simulation time)
  */
-static bool verbose = false;
+static bool kVerbose = false;
```

Comment:
```suggestion
const static bool kVerbose = false;
```

Reply:
Ok. I've changed the other usbdev t-l tests for conformity at the same time, since they've been lagging the coding style.